src/aics-image-viewer/shared/utils/controlPointsToLut.ts

```diff
@@ -51,7 +51,14 @@ export function controlPointsToLut(controlPoints: ControlPoint[]): Uint8Array {
   ctx.fillStyle = grd;
   ctx.fillRect(0, 0, 256, 1);
   const imgData = ctx.getImageData(0, 0, 256, 1);
-  return new Uint8Array(imgData.data.buffer);
+
+  const lut = new Lut();
+  lut.lut = new Uint8Array(imgData.data.buffer);
+  lut.controlPoints = controlPoints;
+
+  // TODO Replace this whole function with new Lut().createFromControlPoints(controlPoints) ?
```

Comment:
Probably easy enough to run a test and compare the results of this function with `createFromControlPoints`. I think in general I'd like to see this happen in the name of less code duplication and because this function's reliance on drawing on a global `canvas` element makes me a bit nervous. (Though maybe it's more efficient than the `for` loops in `createFromControlPoints`? Could run a quick test on that too.) I'm just fine letting this PR through without this change though.

Reply:
Yeah my feeling is this probably should be done but on its own and later.